data_transfer/devices/dreem.py

```diff
@@ -33,46 +33,51 @@ def download_metadata(self) -> None:
         Before downloading raw data we need to know which files to download.
         Dreem provided a range of metadata (including a report) per data record.
 
-        This method downloads and stores the metadata as file, and stores most 
+        This method downloads and stores the metadata as file, and stores most
         relevant metadata as a Record in the database in preparation for next stages.
-        
+
         NOTE/TODO: will run as BATCH job.
         """
-        # Note: includes metadata for ALL data records, therefore we must filter them 
+        # Note: includes metadata for ALL data records, therefore we must filter them
         all_records = dreem_api.get_restricted_list(self.session, self.user_id)
 
-        # Only add records that are not known in the DB based on stored filename (ID and filename in dreem)
-        unknown_records = [r for r in all_records if r['id'] not in set(all_filenames())]
+        # Only add records that are not known in the DB based on stored filename
+        # i.e. (ID and filename in dreem)
+        unknown_records = [
+            r for r in all_records if r["id"] not in set(all_filenames())
+        ]
 
         # Aim: construct valid record (metadata) and add to DB
         for item in unknown_records:
 
-            device_serial = dreem_api.serial_by_device(item['device'])
+            device_serial = dreem_api.serial_by_device(item["device"])
             device_id = inventory.device_id_by_serial(device_serial)
 
             # NOTE: lookup Patient ID by email: if None (e.g. personal email used), then use inventory
-            patient_id = dreem_api.patient_id_by_user(item['user']) or inventory.patient_id_by_device_id(device_id)
-            
+            patient_id = dreem_api.patient_id_by_user(
+                item["user"]
+            ) or inventory.patient_id_by_device_id(device_id)
+
             # NOTE: using inventory to determine intended wear time period.
-            # Useful for historical data, but (TODO) should be replaced with UCAM API. 
+            # Useful for historical data, but (TODO) should be replaced with UCAM API.
             history = inventory.device_history(device_id)[patient_id]
 
-            start_wear = utils.format_weartime(history['checkout'])
+            start_wear = utils.format_weartime(history["checkout"], "inventory")
```

Comment:
@jawrainey , not that `mypy` indicated that a second argument was missing (hence I added "inventory" here and two lines further on). Do let me know if I misinterpreted this.

Reply:
It's correct and is a bug I introduced 👀  Thanks `mypy`!